code/src/components/CustomCheckbox.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState } from 'react';
+
+import styled from 'styled-components';
+
+// A nested component. A function that takes in props and renders an input type = checkbox. 
+// It is rendering a TYPE checkbox and not a STYLED checkbox. Anything we send in as an attribute 
+// below in styled Checkbox, comes out at props - that function Checkbox can use.
+// {...props} = add these props inside the input.
+
+// A nested function? Creates a styled, hidden but accessible checkbox. Accessible
```

Comment:
Super well done implementing a Custom Checkbox - this took extra time and was not super simple to understand, so really well done 🎉

Reply:
Really nice checkbox! A little confusing that it's `cursor: pointer;` everywhere else but on the checkbox though :)

Reply:
Same goes for all other clickable areas